src/components/Article.jsx

```diff
@@ -25,6 +26,33 @@ export default function Article()
         fetchArticles();
     }, []);
 
+    const voteHandler = (change) => async () => {
```

Comment:
Although pretty neat, if you were collaborating on this project this syntax could cause confusion and make bugs more difficult to remedy. It might be better to just have `voteFunction(number)` and in the button's onClick `(()=>voteFunction(num))`.